backend/curriculum_tracking/management/commands/mark_freecodecamp_projects.py

```diff
@@ -0,0 +1,180 @@
+import json
+import os
+
+from playwright.sync_api import sync_playwright, Page
+
+from django.core.management.base import BaseCommand, CommandParser
+from django.utils import timezone
+
+from curriculum_tracking.models import AgileCard, ContentItem, RecruitProjectReview
+from curriculum_tracking.constants import RED_FLAG, NOT_YET_COMPETENT, COMPETENT
+from taggit.models import Tag
+from core.models import User
+from backend.settings import (
+    CURRICULUM_TRACKING_REVIEW_BOT_EMAIL,
+    CURRICULUM_TRACKING_TRUSTED_REVIEW_BOT_EMAIL,
+)
+
+FREECODECAMP_AUTOMARKER_DATA_PATH = os.environ.get(
+    "FREECODECAMP_AUTOMARKER_DATA_PATH", None
+)
+
+
+NYC_TEMPLATE = """Something has gone wrong - your timeline is missing some of the required items. Please make sure you have completed all required sections relevant to this project. You can click on __View Content__ on your project page to see the project instructions and requirements.
+
+The missing items are:
+- {missing_items}
+"""
+RED_FLAG_TEMPLATE = """Something has gone wrong - We couldn't find your "timeline" on freecodecamp. Please make sure you have provided a valid link and all your privacy settings are set to "Public". """
+
+
+NEXT_BTN_SELECTOR = "ul.timeline-pagination_list button[aria-label='Go to next page']"
+
+
+class Command(BaseCommand):
+    def add_arguments(self, parser: CommandParser) -> None:
+        parser.add_argument(
+            "--elevated",
+            action="store_false",
+            help="If the bot should be elevated to a superuser",
+        )
+        parser.add_argument(
+            "--dry-run",
+            action="store_true",
+            help="If the bot should add reviews",
+        )
```

Comment:
I don't get these arguments...

Would it not just be easier to have one that determines whether reviews are left? Maybe `prod` for leaving reviews and not leaving the reviews if the argument is not added? Or the opposite of `dry run`

I think this would also mean the bot is a superuser/trusted when the above argument is added. If not added the user is the bot_user.

Maybe I'm missing something

Reply:
Initially, the automarker would add competent but untrusted reviews, prompting a human review to assess its efficacy. After verifying its competence, we’d run it as super to automatically close cards. NCIT cur marker does this, but it differs. I’ve reconsidered and believe we won’t need it.